### ISSUE TITLE
Edit excel and Save same excel in same path

### ISSUE THREAD

**Issue by mahesh-s-nikam** [abap2xlsx/abap2xlsx#454]
hello,

We are trying to use ABAP2XLSX technology for editing excel and save same excel file at same path(from where we have read that excel).

We are editing excel sheet by adding some values in certain cells, then saving same excel at same path.
This process is happening good. 

But, when we open that excel file we get attached error
![error when open excel file](https://cloud.githubusercontent.com/assets/20464002/18088215/ea94463a-6ed6-11e6-9c61-f83eeff76493.JPG)

Or some times we donot get such error and we are able to open same excel file. 
But when we are closing that excel file without editing any part of this excel, we get pop up of save or don't save.

![image](https://cloud.githubusercontent.com/assets/20464002/18088580/0b2bbb2e-6ed9-11e6-9e33-5ecbc1d7ddfa.png)

can anyone help us to resolve this issue?

Regards,
Mahesh Nikam.

**Comment by gregorwolf** [comment]
Dear Mahesh,

seems the first issue occurs with specific constellations in the data you're passing to the excel. Perhaps you can provide a non confidential example that we can investigate.

Regarding the location you're storing the file to can you describe us the whole process please. Which report stores the data where? What should happen with the changed file?

Best regards
Gregor

**Comment by mahesh-s-nikam** [comment]
Hello,

Sorry for late reply. We were trying to resolve this but did not resolved from us.
This is attached prog Z_EDIT_EXCEL
[Z_EDIT_EXCEL.txt](https://github.com/ivanfemia/abap2xlsx/files/451660/Z_EDIT_EXCEL.txt)

And this is attached excel file which we are uploading in above program for editing and saving.
[POC getting excel from SAP - original.xlsx](https://github.com/ivanfemia/abap2xlsx/files/451662/POC.getting.excel.from.SAP.-.original.xlsx)

Once we upload this attached excel, we are editing sheet 'matl master data'.
In this 'matl master data' sheet, we are using material value from column Material (col A), and we are filling values in below 2 corresponding columns 
- column safety stock(col B) - filling value from table field MARC-EISBE (of any plant )
- column base uom (col C) - filling value from table field MARA-MEINS

Can you help us to resolve issue as mentioned in 1st comment.

Regards,
Mahesh Nikam.

**Comment by mahesh-s-nikam** [comment]
can anyone please help us to resolve this issue?

**Comment by StefanSchmoecker** [comment]
Dear Mahesh Nikam,

Gregor asked you to provide a non-confidential example of such an erronous file.  We'd like to help you but you'd have to listen to the suggestions you receive.
I did look into your coding and tried it on my system. Apart from the fact that it is no felony to use comments or modularization in ABAP I was able to get it to work with some testdata and it came out just fine.  
The only thing left for me to do is to reinfore Gregor's suggestion.

---

live long and prosper
Stefan Schmöcker
